### ISSUE TITLE
Using new API in CakePHP

### ISSUE THREAD

**Issue by chicagosky** [cloudconvert/cloudconvert-php#10]
Hello

First of all thanks for a great utility. I can't tell you how much I use it and love it. I am integrating with your service from a PHP application that is built using CAKEPHP 2.5.4

To begin with I used your legacy CloudConvert.class.php and it worked wonderfully and still continues to do so. However, I noticed you have published a newer version that uses Guzzle etc. I am unfamiliar with Guzzle and also when I try and include your classes in my application I get some compilation errors in Api.php, ApiObject.php etc. 

Do you have any advice on 
1. How to integrate seamlessly with CakePHP?
2. If I do intend to use the newer version if I should download Guzzle as well?

**Comment by josiasmontag** [comment]
Hi,

Installing it with CakePHP should be pretty easy, if your CakePHP installation was also done with composer: http://book.cakephp.org/2.0/en/installation/advanced-installation.html. In this case you only need to add ``"cloudconvert/cloudconvert-php": "dev-master"``to your ``require``section in ``composer.json``. Using ``php composer.phar install`` the wrapper will be automatically installed, including all dependencies (Guzzle).

If you did not use composer for your CakePHP installation, I would recommend to download the ``.phar``file from the "Releases" tab and include it as described in the Readme. You do not need to download Guzzle seperately, it is included in the .phar file.

Hope that helps.

**Comment by chicagosky** [comment]
Ok - I tried doing what you said. Ran into a few issues

1. Including the Phar file as-is didnt work for some reason. I got a lot of errors at runtime
2. So I exploded the Phar file and included the below statements in my PHP file (using CakePHP convention) - I still get the error that Api class is not recognized. 

Any help is much appreciated

<code>App::import('Vendor', 'CloudConvert', array('file' => 'cloudconvert/vendor/autoload.php'));
App::import('Vendor', 'Api', array('file' => '\CloudConvert\Api'));</code>
